### ISSUE TITLE
ROI pooling layer?

### ISSUE THREAD

**Issue by cloudtoro** [smallcorgi/Faster-RCNN_TF#23]
Hi,

I am wondering if this implementation include any ROI Pooling Layer?

My understanding is the ROI Pooling is not supported in generic TensorFlow. I could be wrong though...

Thanks!
cloud

**Comment by Yuliang-Zou** [comment]
Yes, this implementation includes ROI Pooling layer, which is a necessary part of the Faster-RCNN.

You can find in in ./lib/roi_pooling_layer/ folder.
